src/github.com/getlantern/rot13/rot13_test.go

```diff
@@ -0,0 +1,39 @@
+package rot13
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRoundTrip(t *testing.T) {
+	var in, out, rt bytes.Buffer
+	for i := byte(0); i <= 255; i++ {
+		err := in.WriteByte(i)
+		if !assert.NoError(t, err, "Unable to write byte") {
+			return
+		}
+		if i == 255 {
```

Comment:
Because I wanted to test all 256 possible values of a byte.  If I don't break at 255, i++ brings wraps back to 0, which meets the boundary condition, so this becomes an infinite loop.